src/locale/LangKey.ts

```diff
@@ -4,8 +4,8 @@ enum LangKey {
   AMOUNT_NOT_NUMERIC = 'AMOUNT_NOT_NUMERIC',
   AMOUNT_TOO_HIGH = 'AMOUNT_TOO_HIGH',
   BEKREFT = 'BEKREFT',
-  BEKREFT_LABEL = 'BEKREFT_LABEL',
-  BEKREFT_OPPLYSNINGER = 'BEKREFT_OPPLYSNINGER',
+  BEKREFTOPPLYSNINGER_BEKREFT_LABEL = 'BEKREFTOPPLYSNINGER_BEKREFT_LABEL',
+  BEKREFTOPPLYSNINGER_BEKREFT_OPPLYSNINGER = 'BEKREFTOPPLYSNINGER_BEKREFT_OPPLYSNINGER',
```

Comment:
Synes det er lettere å lese den forrige nøkkelen.
Tenkte ikke over det når du viste meg, men er det en grunn til at navnet er så langt?

Reply:
For å gjøre tydligere hvor språknøklene hører hjemme så ble vi enige om å bruke komponentets navn først etterfulgt av den "lokale" verdien til slutt. F.eks KOMPONENT_FELTNAVN. Når jeg ser på det nå så kunne vi ha brukt: BEKREFTOPPLYSNINGER_LABEL og ikke BEKREFTOPPLYSNINGER_BEKREFT_LABEL. Disse verdiene ligger nå i felles så vi kan vurdere om vi skal endre de der....?